src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,199 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tale;
+    private int size = 0;
+
     @Override
     public void add(T value) {
+        if (size >= 1) {
+            Node<T> prevTale = tale;
+            tale = new Node(prevTale, value, null);
+            prevTale.next = tale;
+        } else if (size == 0) {
```

Comment:
```suggestion
        } else {
```
size can't be negative, so no need in else if